### ISSUE TITLE
Is it possible to start node-webkit as the window manager?

### ISSUE THREAD

**Issue by CoryGH** [nwjs/nw.js#722]
Is it possible to start node-webkit as the window manager?  So that instead of having the overhead of a window manager you just boot into a single site?

I'm trying to use a bunch of Raspberry Pis as stations to automate away the paperwork for a production line - the tracking software is already designed for node-webkit but the combination of not wanting people to fiddle with the OS or have wasted screenspace (this part could be accomplished with kiosk mode) and wanting to keep CPU overhead to a bare minimum leads me to ask this question.

TL;DR: I want to start a single-window node-webkit app in kiosk mode that replaces the desktop window manager entirely.
##

**Comment by rduckett** [comment]
How can I get gdm/entrance/kdm/etc to do this? Infact, what pertains to getting a complete NW.JS system running (at least in linux). I may have a look at combining nwm and NW.JS just to see if the bindings are all compatible and if I can have desktop apps running in HTML elements!
